### DISCUSSION TITLE
Welcome to add Discussions!

### DISCUSSION THREAD

**Discussion by artbear** [vanessa-opensource/add#822] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sulfur17** [comment]
@artbear какой должен быть порядок запуска стандартных процедур в тесте?
Вот сейчас, например, если в обработке есть набор из двух тестов, то для каждого теста дважды запускается Инициализация и по одному разу ПередЗапускомТеста - ПослеЗапускаТеста.
Т.е. для набора из двух тестов Инициализация запускается четырежды, а ПередЗапускомТеста дважды.
А как должно быть?

**Reply by artbear** [reply]
@sulfur17 должно быть (1 + 1) * 2 (теста)

но из-за особенностей работы серверных тестов в упр.режиме, т.к. у них нет состояния, приходится Инициализацию вызывать каждый раз.

для оперативности можем в чате телеграма обсудить. https://t.me/vanessa_opensource_chat

**Reply by artbear** [reply]
и поэтому (2 иниц + 1 передЗапуском) * 2 теста (

**Comment by PLebedevV** [comment]
@artbear Возможно ли использование bddRunner.epf под пользователем с неполными правами в 1С? Хотим что бы пользователь генерировал тесты, но не хотим что бы у него были полные права.

**Comment by SergeyBoev** [comment]
Прогоняю стандартный стандартные дымовые тесты командой vrunner xunit
--------- venv.json -----------------
"xunit": {
"--xddConfig": "./tools/JSON/vanessaTddConf.json",
"testsPath": "$addRoot/tests/smoke",
"--reportsxunit": "ГенераторОтчетаJUnitXML{out/smoke/junit/junit.xml};ГенераторОтчетаAllureXMLВерсия2{out/smoke/allure/allure.xml}",
"--xddExitCodePath": "./build/xddExitCodePath.txt"
},

Вылетает много ошибок с открытием неинтерактивных форм

{ВнешняяОбработка.xddTestRunner.Форма.УправляемаяФорма.Форма(270)}:АСинк().смв_НайтиФайлы(ОбработчикПоискаКлиентскихПлагинов,
{ВнешняяОбработка.xddTestRunner.Форма.УправляемаяФорма.Форма(71)}:ЗагрузитьПлагины();

[ОшибкаВоВремяВыполненияВстроенногоЯзыка]
по причине:
Недоступный пользователю элемент управления не может выполнять интерактивные действия

Не могу понять как исключить эти формы из теста
Хотелось бы все скопом.
Полистал [документацию](https://forum.infostart.ru/redirect.php?url=aHR0cHM6Ly9naXRodWIuY29tL3ZhbmVzc2Etb3BlbnNvdXJjZS9hZGQvYmxvYi9kZXZlbG9wL3Rlc3RzL3Ntb2tlL3JlYWRtZS5tZCPt4PHy8O7p6uAt5Pvs7uL79S3y5fHy7uIt9O7w7C3u4frl6vLu4i3v7uQt6u7t6vDl8u3z/i3q7u306OPz8OD26P4=) не помогло
Понял, что конфигурируется в vanessaTddConf.json секция "smoke": {
Но как конкретно, не по имени же. Поделитесь образцом
